test/lit.cfg

```diff
@@ -1292,6 +1292,8 @@ config.substitutions.append(('%scale-test',
                                  config.scale_test, config.swiftc)))
 config.substitutions.append(('%empty-directory\(([^)]+)\)',
                              SubstituteCaptures(r'rm -rf "\1" && mkdir -p "\1"')))
+config.substitutions.append(('%check-file-exists\(([^)]+)\)',
+                             SubstituteCaptures(r'''%r -c 'import os; exit(0) if os.path.exists("\1") else exit(1)' ''' % sys.executable)))
```

Comment:
This is an ugly way check if a file exists without using any of the gnu core utils which don't support utf16